benchmarks/experiment_runner.py

```diff
@@ -232,6 +233,20 @@ def prepare_inputs(self, example_inputs, should_randomize_input):
       inputs_list.append(inputs)
     return inputs_list
 
+  def dump_profile_info(self, prof, model_name):
+    assert prof is not None, 'Expecting profiler to be defined!'
+    if not self._args.profile_cuda_dump:
```

Comment:
what do you think we also return an error here, just so users wouldn't wonder where the dump goes?

Reply:
I can add additional logging, but returning an error does not make much sense, as it is not an error really.

Reply:
If the user enables `--profile-cuda`, is the dump the only place where the user can see the profile?